static/app/actionCreators/navigation.spec.tsx

```diff
@@ -120,4 +120,22 @@ describe('navigation ActionCreator', () => {
     expect(openModal).not.toHaveBeenCalled();
     expect(router.push).toHaveBeenCalledWith('/settings/projects/project-slug/alerts/');
   });
+
+  it('preserves query parameters in path object', function () {
+    router.location.query.project = '2';
+    expect(
+      navigateTo(
+        {
+          pathname: '/settings/:projectId/alert?project=:project#performance-sidequest',
+          query: {referrer: 'onboarding_task'},
+        },
+        router
+      )
+    ).toBe(undefined);
```

Comment:
`navigateTo` is a `void` function, so it is always `undefined`. Do we need to have these checks?

Reply:
i'm not sure why that's there i did copy and paste this from another test